### PR TITLE
[Releasenotes] GraFx Platform API 1.14.0

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

**Review by tsudd — APPROVED** [review]
Looks great, thanks!

**Comment by tsudd** [comment]
Sean is right, we can take it over and prepare release notes by our own.
We will follow the process described in [CONTRIBUTING.MD](https://github.com/chili-publish/grafx-documentation/blob/main/CONTRIBUTING.md).
Hope it's fine

File: docs/release-notes/index.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ CHILI GraFx [Environment API 1.1.9](https://sandbox1.chili-publish-sandbox.onlin
 
 ![releasenotes](https://github.com/chili-publish/grafx-documentation/assets/122599725/b0b9bb66-1f25-4c70-bd95-e4e0c7e94f74)
 
+CHILI GraFx [Platform API](https://api.chiligrafx.com/swagger/index.html)
+
+### Improvements
+
+- GET /subscription/id and /subscription/id/renders were migrated to use GUIDs
```
Comment (jeroencosyn) [inline docs/release-notes/index.md:78]: There's a punctuation mark missing after 'GUIDs'. The next line is just concatenated now.

File: docs/release-notes/index.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ CHILI GraFx [Environment API 1.1.9](https://sandbox1.chili-publish-sandbox.onlin
...
+Now all our end points use GUIDs for identifying entities
```
Comment (jeroencosyn) [inline docs/release-notes/index.md:79]: Here and in the next line you write 'end point' but everywhere else we write 'endpoint'.

**Pull request merged**